.github/workflows/format-native.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        machine: [ windows-2022, ubuntu-22.04, macos-13-xlarge ]
+        machine: [ windows-2025, ubuntu-22.04, macos-13-xlarge ]
```

Comment:
note: seems here all images could use the latest version

Reply:
#4266